Libs/MRML/Core/vtkMRMLTransformDisplayNode.cxx

```diff
@@ -88,6 +88,12 @@ vtkMRMLTransformDisplayNode::vtkMRMLTransformDisplayNode()
     this->ContourLevelsMm.push_back(level);
     }
 
+  this->EditorVisibility = 0;
+  this->EditorSliceIntersectionVisibility = 0;
+  this->TranslationEnabled = 1;
```

Comment:
It would be better to prefix these with "Editor":
this->EditorTranslationEnabled
this->EditorRotationEnabled
this->ScalingEnabled

Also, use bool type for Boolean variables and true/false to set their value (earlier it was a limitation of the TCL wrapper that it could not handle bool type, but this limitation does not exist anymore)